.idea/codeStyles/Project.xml

```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please remove this file from your commit history. These changes must have come from android studio automatically.

Reply:
This file is still here, we need to revert this entire file.